misc/fs10.0.0up.txt

```diff
@@ -881,7 +886,7 @@ or
   http://www.metsahovi.fi/fs/dist/old/
 
 If you have a version older than 9.3.13, please contact Ed
-(Ed.Himwich@nasa.gov).
+(Ed.Himwich@nviinc.com).
```

Comment:
(This is mostly testing out comments/code review) @wehimwich what's the status of your NASA email?

Reply:
still is use, but maybe nviinc is the one to use in the document